validation-test/stdlib/Bool.swift

```diff
@@ -0,0 +1,45 @@
+// RUN: mkdir -p %t
+// RUN: %target-clang -x c %S/Inputs/VariadicBool/variadicBool.c -c -o %t/variadicBool.o
+// RUN: %target-build-swift -I %S/Inputs/VariadicBool/ %t/variadicBool.o %s -parse-stdlib -Xfrontend -disable-access-control -o %t/a.out
```

Comment:
Do we really need these `-parse-stdlib` and `-Xfrontend -disable-access-control` arguments?

Reply:
Oh right, I added those to access the `value` property in earlier versions of the code, not needed now.